### PR TITLE
LPS-140939 As a remote System Admin, I can set the email and name of the admin when creating a virtual instance

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-140939](https://issues.liferay.com/browse/LPS-140939).<br/><br/>/cc @pei-jung @alee8888 @ChrisKian @patriciajeanperez @lr-leo @erickmonteiroo

@epgarcia will you please look over these changes and let me know if you'd like anything changed? Thank you!

### CODE REVIEW

**Comment by epgarcia** [comment]
Hi @drewbrokke. It looks good to me, thanks! @ZoltanTakacs FYI

**Comment by drewbrokke** [comment]
force-pushed a change to include a missing companyId.

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
ci:test:sf

File: modules/apps/headless/headless-portal-instances/headless-portal-instances-impl/src/main/java/com/liferay/headless/portal/instances/internal/resource/v1_0/PortalInstanceResourceImpl.java [hunk 1]
```diff
@@ -101,14 +110,38 @@ public PortalInstance patchPortalInstance(
...
+				company.getCompanyId(), administratorEmailAddress,
+				administratorFirstName, administratorLastName);
+
+			User defaultAdminUser = _userLocalService.getUserByEmailAddress(
+				company.getCompanyId(),
+				PropsValues.DEFAULT_ADMIN_EMAIL_ADDRESS_PREFIX + "@" +
+					company.getMx());
+
+			defaultAdminUser.setScreenName(
+				administratorFirstName + administratorLastName);
```
Comment (pei-jung) [inline modules/apps/headless/headless-portal-instances/headless-portal-instances-impl/src/main/java/com/liferay/headless/portal/instances/internal/resource/v1_0/PortalInstanceResourceImpl.java:137]: I think we can use the `ScreenNameGenerator` to get the screen name out of the email address.

Comment (drewbrokke) [reply modules/apps/headless/headless-portal-instances/headless-portal-instances-impl/src/main/java/com/liferay/headless/portal/instances/internal/resource/v1_0/PortalInstanceResourceImpl.java:137]: Good catch! I'll update that and force-push.

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**